src/components/build/investigationBuildPage/investigationBuildPage.tsx

```diff
@@ -98,13 +98,34 @@ const InvestigationBuildPage: React.FC<InvestigationBuildProps> = props => {
         }
       }
     }
-  }, [props.brick]);
+  }, [props.brick, brickId]);
   /* Synthesis */
 
   if (!props.brick) {
     return <div>...Loading...</div>;
   }
 
+  const canEditBrick = (brick: any, user: User) => {
```

Comment:
There is something screwy here that needs to be resolved. Builders cannot alter their own bricks